test/unittests/cpp_test.cpp

```diff
@@ -111,10 +111,8 @@ TEST(cpp, std_hash)
 #pragma warning(disable : 4307 /* integral constant overflow */)
 #pragma warning(disable : 4309 /* 'static_cast': truncation of constant value */)
 
-#if !defined(_MSC_VER) || (_MSC_VER >= 1910 /* Only for Visual Studio 2017+ */)
     static_assert(std::hash<evmc::address>{}({}) == static_cast<size_t>(0xd94d12186c0f2fb7), "");
     static_assert(std::hash<evmc::bytes32>{}({}) == static_cast<size_t>(0x4d25767f9dce13f5), "");
```

Comment:
Why have both `static_assert` and `EXPECT_EQ` for the same cases here?

Reply:
The `constexpr` functions only mean "may be executed in constexpr context". But they can also be executed with non-constexpr arguments at runtime. Additional runtime executions also contribute to code coverage.